src/mlagility/helpers/labels.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Dict, List
+
+
+def load_from_file(file_path: str) -> Dict[str, List[str]]:
+    """
+    This function extracts labels from a Python file.
+    Labels must be in the first line of a Python file and start with "# labels: "
```

Comment:
Update #24 with any rules like this that need to be documented in the `models/readme.md`